### ISSUE TITLE
remove placeholder in output

### ISSUE THREAD

**Issue by bwallace** [gdietz/OpenMEE#206]

**Comment by gdietz** [comment]
@bwallace Need more info.

**Comment by bwallace** [comment]
why? just remove it?

On Sun, May 8, 2016 at 9:13 PM, George Dietz <notifications@github.com>
wrote:

> @bwallace <https://github.com/bwallace> Need more info.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/gdietz/OpenMEE/issues/206#issuecomment-217762715>
>



-- 
byron

**Comment by gdietz** [comment]
@bwallace What are we talking about? What placeholder in what output?

**Comment by JGurevitch** [comment]
The  'placeholder' at the bottom of the output of a meta-analysis AND
meta-regression should be deleted (see screenshot below). Much better would
be to replace it with a citation to the Handbook: Koricheva, J., J.
Gurevitch and K. Mengersen, eds. 2013. *Handbook of Meta-analysis in
Ecology and Evolution*. Princeton Univ. Press, Princeton, NJ
[image: Inline image 1]

On Wed, May 11, 2016 at 5:30 AM, George Dietz <notifications@github.com>
wrote:

> @bwallace <https://github.com/bwallace> What are we talking about? What
> placeholder in what output?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gdietz/OpenMEE/issues/206#issuecomment-218408483>
>



-- 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Jessica Gurevitch
Professor
Department of Ecology and Evolution
Stony Brook University
Stony Brook, NY 11794-5245 USA
+1-631-632-8567
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by gdietz** [comment]
@JGurevitch Can you try reattaching the screenshot?

**Comment by JGurevitch** [comment]
Go to the ouput from a meta-analysis or meta-regression and scroll to the
bottom.

On Thu, May 12, 2016 at 2:39 AM, George Dietz <notifications@github.com>
wrote:

> @JGurevitch <https://github.com/JGurevitch> Can you try reattaching the
> screenshot?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/gdietz/OpenMEE/issues/206#issuecomment-218672812>
>



-- 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Jessica Gurevitch
Professor
Department of Ecology and Evolution
Stony Brook University
Stony Brook, NY 11794-5245 USA
+1-631-632-8567
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

**Comment by bwallace** [comment]
I have removed the reference sections throughout and updated the unit tests to expect this reduced output. These still pass on my end. 

@gdietz please verify then close?

**Comment by JGurevitch** [comment]
The output should report what was tested (that is, what the effect size
was). Sometimes you are testing several different effect sizes on a data
set and the output doesn't record what was tested (assuming they have
different names). Anything else that was chosen in the analysis should also
be reported in the output (e.g. random effects model using REML).

On Thu, Jun 9, 2016 at 7:09 PM, George Dietz <notifications@github.com>
wrote:

> Closed #206 <https://github.com/gdietz/OpenMEE/issues/206>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/gdietz/OpenMEE/issues/206#event-687948396>, or mute
> the thread
> <https://github.com/notifications/unsubscribe/AGJ4umy-6AhDgIehySWiBCQjujlNkT7iks5qKJ0agaJpZM4IP9Xn>
> .
>



-- 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Jessica Gurevitch
Professor
Department of Ecology and Evolution
Stony Brook University
Stony Brook, NY 11794-5245 USA
+1-631-632-8567
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
